docs/data/toolpad/core/components/use-dialogs/use-dialogs.md

```diff
@@ -0,0 +1,104 @@
+---
+productId: toolpad-core
+title: useDialogs
+components: DialogsProvider
+---
+
+# Dialogs
+
+<p class="description">Imperative APIs to open and interact with dialogs.</p>
+
+Toolpad core offers a set of abstractions that makes interacting with dialogs simpler. It has an imperative API to open and close dialogs, and allows dialogs to be stacked on top of each other.
+
+First thing you need to do is install the DialogsProvider at the root of your application.
+
+```tsx
+import { DialogsProvider } from '@toolpad/core/useDialogs';
+
+function App({ children }) {
+  return <DialogsProvider>{children}</DialogsProvider>;
```

Comment:
How common is it going to be that someone has to import the provider directly instead of just using `AppProvider`?
That's why so far I haven't documented the branding and navigation providers like this, but maybe it could be done for consistency, or maybe dialogs and navigation might have more specific use cases than those?

Reply:
The notifications and dialogs can stand perfectly on their own. The branding and navigation providers are just implementation details of the app provider. Internal until tehre is a good reason not to.